packages/types/src/lo-hi.ts

```diff
@@ -34,9 +34,9 @@ export const splitLoHi = (value: bigint): Required<LoHi> => {
  * returns a bigint that combines these two parts.
  * To achieve this, we shift `hi` 8 bytes to the left.
  *
- * @param {bigint} lo - The low order 64 bits of the number.
- * @param {bigint} hi - The high order 64 bits of the number.
```

Comment:
Not in scope for this PR, but these `@param` declarations should be moved to `/** ... */`-style comments before the params themselves in the function definition. JSDoc style comments aren't needed in TypeScript land :)